src/Dapr.AzureFunctions.Extension/Triggers/DaprTopicTriggerAttribute.cs

```diff
@@ -16,8 +16,32 @@ namespace Dapr.AzureFunctions.Extension
     public class DaprTopicTriggerAttribute : Attribute
     {
         /// <summary>
-        /// Gets or sets the name of the topic.
+        /// Initializes a new instance of the <see cref="DaprTopicTriggerAttribute"/> class.
+        /// </summary>
+        /// <param name="pubSubName">The pub/sub name.</param>
+        public DaprTopicTriggerAttribute(string pubSubName)
+        {
+            if (pubSubName is null)
+            {
+                throw new ArgumentNullException(nameof(pubSubName));
+            }
+
+            this.PubSubName = pubSubName;
+        }
+
+        /// <summary>
+        /// Gets the pub/sub name.
```

Comment:
Don't see the comment in the doc here.

Reply:
Which comment? - I'm not sure what you're asking for sorry 😬

Reply:
Never mind....it was from my limited understanding...